data/tn/legislature/Torrey-C-Harris-7b19a0d5-6f61-471b-8869-bfd249f3d6fd.yml

```diff
@@ -0,0 +1,31 @@
+id: ocd-person/7b19a0d5-6f61-471b-8869-bfd249f3d6fd
+name: Torrey C Harris
+email: rep.torrey.harris@capitol.tn.gov
+party:
+- name: Democratic
+roles:
+- type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:tn/government
+  district: '91'
```

Comment:
@sroomf this person should have a file already, we just need to add their new role:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:tn/government
  district: '91'
They were previously in house district 90 from November 3, 2020 until November 8, 2022